docs/CLIReadme.md

```diff
@@ -510,10 +510,9 @@ at the host zos124 (default port - 443) and allow self-signed certificates:
 
       * `$  zowe profiles create zosmf-profile zos124 --host zos124 --user ibmuser --password myp4ss --reject-unauthorized false`
 
-*  Create a zosmf profile called 'zos124' to connect to z/OSMF
-at the host zos124 (default port - 443) and allow self-signed certificates:
+*  Create a zosmf profile called 'zosAPIML' to connect to z/OSMF via the Zowe API Mediation Layer running at host 'zosAPIML', port '60004', and allow for self-signed certificates. To reduce duplication, you could elect to store the 'host', 'port', 'reject-unauthorized', 'user', and 'password' values for the API Mediation Layer in a base profile and only store the 'base-path' of the service in the zosmf profile.:
```

Comment:
This readme is automatically generated, so the example description here should match the description in `imperative.ts`.